static/app/views/eventsV2/table/index.tsx

```diff
@@ -195,7 +191,7 @@ class Table extends PureComponent<TableProps, TableState> {
   }
 }
 
-export default withApi(withTags(Table));
+export default withApi(Table);
```

Comment:
since `withTags` is being used in several other places, do you think we can create a reusable hook `useTags`?

Reply:
@priscilawebdev I've added a barebones implementation of `useTags()`.